app/views/Settings/ImportFromUrl.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react';
+import {
+  StyleSheet,
+  View,
+  TextInput,
+  Alert,
+  NativeModules,
+} from 'react-native';
+import { useTranslation } from 'react-i18next';
+import importLocationsApi from '../../api/import/importLocationsApi';
+
+import { NavigationBarWrapper } from '../../components/NavigationBarWrapper';
+
+import { Typography } from '../../components/Typography';
+import { Button } from '../../components/Button';
+import { NavigationProp } from '../../navigation';
+
+import { Forms } from '../../styles';
+
+type ImportFromUrlProps = {
+  navigation: NavigationProp;
+};
+
+const ImportFromUrl = ({ navigation }: ImportFromUrlProps): JSX.Element => {
+  const { t } = useTranslation();
+  const [url, setUrl] = useState('');
+
+  // TODO: add logic to filter mocked data. We should agree on flag name
+
+  const importData = async () => {
+    try {
+      // Data must be an array of objects!
+      const mockData = await importLocationsApi(url);
+
+      if (mockData) {
+        await NativeModules.SecureStorageManager.importGoogleLocations(
+          mockData,
+        );
+        return navigation.goBack();
+      }
+      throw new Error();
+    } catch (e) {
+      return Alert.alert(t('import.mockData.invalid_url'));
```

Comment:
Can this also alert the error message?

```suggestion
      return Alert.alert(t('import.mockData.invalid_url'), e.message);
```